File: RukaLjubavi.Api/Models/OcjenaDonacije.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace RukaLjubavi.Api.Models
{
    public enum TipKorisnika
    {
        Donator = 1,
        Benefiktor = 2
    }
    public class OcjenaDonacije : IEntity
    {
        public int Id { get; set; }
        public string Komentar { get; set; }
        public int Povjerljivost { get; set; }
        public int BrzinaDostavljanja { get; set; }
        public int PostivanjeDogovora { get; set; }
        public TipKorisnika Ocjenjivac { get; set; }
        public int KorisnikId { get; set; }

        [ForeignKey(nameof(DonacijaId))]
        public Donacija Donacija { get; set; }
        public int DonacijaId { get; set; }
    }
}

File: RukaLjubavi.Android/app/src/main/java/com/fit/ba/rukaljubavi/Models/OcjenaDonacije.kt
package com.fit.ba.rukaljubavi.Models

import java.io.Serializable

class OcjenaDonacije(
    var id: Int,
    var komentar: String,
    var povjerljivost : Int,
    var brzinaDostavljanja : Int,
    var postivanjeDogovora : Int,
    var korisnikId  : Int,
    var donatorIme : String,
    var donatorPrezime: String,
    var benefiktorNazivKompanije : String
) : Serializable